docs/ldml/tr35.html

```diff
@@ -2369,6 +2369,29 @@ <h4><a href="#Key_And_Type_Definitions_" name=
           <em>Unknown or Invalid Currency</em>.</p>
         </td>
       </tr>
+      <tr class="changed">
+        <td colspan="4"><strong>A <a href=
+        "#UnicodeDictionaryBreakExclusionIdentifier" name=
+        "UnicodeDictionaryBreakExclusionIdentifier" id=
```

Comment:
The anchor names (id=...) in the spec use a variety of styles, but underscores seem common. Do we care?

Reply:
At least in this table, id values specified as a category are italicized, while explicit values are quoted and non-italic. So this follows that style.

Reply:
I am talking about the anchor text id="UnicodeDictionaryBreakExclusionIdentifier", not the CSS style of the presentation.
Other anchors use underscores (id="Unicode_Dictionary_Break_Exclusion_Identifier"), lowercase, etc.

Reply:
The camel-case style seems to be used for all of the other id anchors in this table, so I just followed that... e.g.
```
#UnicodeCurrencyFormatIdentifier
#UnicodeCollationIdentifier
#UnicodeCurrencyIdentifier
#UnicodeEmojiPresentationStyleIdentifier
#UnicodeFirstDayIdentifier
...
```